src/app/shared/NoteCreateInline.svelte

```diff
@@ -48,26 +53,24 @@
   }
 
   const onSubmit = async ({skipNsecWarning = false} = {}) => {
+    if ($editorLoading) return
+
     saving = true
 
-    const content = compose.parse().trim()
+    const content = editor.getText().trim()
 
     if (!content) return showWarning("Please provide a description.")
 
     if (!skipNsecWarning && content.match(/\bnsec1.+/)) return nsecWarning.set(true)
 
-    const tags = [...tagsFromContent(content), ...getClientTags()]
+    const tags = [...tagsFromContent(content), ...getClientTags(), ...editor.commands.getMetaTags()]
```

Comment:
Seems like `tagsFromContent` could be replaced by nostr-editor's new tags functionality, or at least `getEditorTags`

Reply:
Cesar recently started working on it, it probably isn't ready yet.
getEditorTags will gives you all #whatever in the editor. It refers to hashtags not nostr event tags